Docs: Mc: fix version in setup guide (#873)
ArchipelagoMW/Archipelago: worlds/minecraft/docs/minecraft_en.md

### Connect to the MultiServer

> Open Minecraft go to multiplayer option and then connect to the server `localhost`.

Reviewer: ```suggestion
Open Minecraft, go to `Multiplayer > Direct Connection`, and join the `localhost` server address.
```
Suggesting an update to use the exact language Minecraft uses.

If you are using the website to host the game then it should auto-connect to the AP server without the need to `/connect`

otherwise once you are in game type `/connect <AP-Address> (Port) (Password)` where `<AP-Address>` is the address of the

> Archipelago server. `(Port)` is only required if the Archipelago server is not using the default port of 38281. In order to type in the port use a space instead of a colon

Reviewer: ```suggestion
Archipelago server. `(Port)` is only required if the Archipelago server is not using the default port of 38281. Note that there is no colon between `<AP-Address>` and `(Port)`.
```
Makes the language clearer to a first-time AP user, without sacrificing meaning.
